examples/fidelity.py

```diff
@@ -0,0 +1,109 @@
+"""Simulates the fidelity estimation.
+
+Direct Fidelity Estimation from Few Pauli Measurements
+https://arxiv.org/abs/1104.4695
+
+Practical characterization of quantum devices without tomography
+https://arxiv.org/abs/1104.3835
+"""
+
+import cirq
+import heapq
+import itertools
+import math
+import numpy
+
+
+def build_circuit():
+  qubits = [cirq.LineQubit(i) for i in range(3)]
+
+  rot0 = cirq.ZPowGate(exponent=0.250)  # T-Gate, non Clifford.
+  rot1 = cirq.XPowGate(exponent=0.123)
+  rot2 = cirq.XPowGate(exponent=0.456)
+
+  circuit = cirq.Circuit()
+  circuit.append(rot0(cirq.LineQubit(0)))
+  circuit.append(rot1(cirq.LineQubit(1)))
+  circuit.append(rot2(cirq.LineQubit(2)))
+
+  return circuit, qubits
+
+
+def build_noisy_circuit(circuit, qubits):
+  noisy_circuit = circuit.copy()
```

Comment:
`return circuit.with_noise(cirq.amplitude_damp(0.01))`

(You will have to merge master into your branch for this to work.)

Reply:
Done (but directly called in the main function). Also merged and verified that the code still runs.